docs/pages/enterprise/sso/ssh-adfs.mdx

```diff
@@ -125,29 +126,15 @@ spec:
 
 This role declares:
 
-* Devs are only allowed to login to nodes labelled with `access: relaxed` label.
-* Developers can log in as `ubuntu` user
-* Notice `{% raw %}{{external["http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname"]}}{% endraw %}` login. It configures Teleport to look at
-  _"http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname"_ ADFS claim and use that field as an allowed login for each user.
-  Also note the double quotes (`"`) and square brackets (`[]`) around the claim name - these are important.
+* Developers are only allowed to login to nodes labelled with `access: relaxed` label.
+* All developers can log in using the `ubuntu` username.
+* In addition, each developer is allowed to use the login derived from their _"http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname"_ ADFS claim.
+  This is configured by the `{% raw %}{{external["http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname"]}}{% endraw %}` login entry.
```

Comment:
We don't need it in this case, I think.

Current logic is this: If variable name inside `{{...}}` exists in `config.json` then it will be replaced, if it does not exist it will be lest as is. So `raw` tag is only needed if you want to escape variable name from the config file.